src/Algebra/Properties/Group.agda

```diff
@@ -96,13 +121,22 @@ identity-unique {x} id = identityˡ-unique x x (proj₂ id x)
 
 inverseˡ-unique : ∀ x y → x ∙ y ≈ ε → x ≈ y ⁻¹
 inverseˡ-unique x y eq = begin
-  x              ≈⟨ left-helper x y ⟩
+  x              ≈⟨ rightDividesʳ y x ⟨
   (x ∙ y) ∙ y ⁻¹ ≈⟨ ∙-congʳ eq ⟩
        ε  ∙ y ⁻¹ ≈⟨ identityˡ (y ⁻¹) ⟩
             y ⁻¹ ∎
 
 inverseʳ-unique : ∀ x y → x ∙ y ≈ ε → y ≈ x ⁻¹
 inverseʳ-unique x y eq = begin
-  y       ≈⟨ sym (⁻¹-involutive y) ⟩
-  y ⁻¹ ⁻¹ ≈⟨ ⁻¹-cong (sym (inverseˡ-unique x y eq)) ⟩
+  y       ≈⟨ ⁻¹-involutive y ⟨
+  y ⁻¹ ⁻¹ ≈⟨ ⁻¹-cong (inverseˡ-unique x y eq) ⟨
   x ⁻¹    ∎
+
+\\≗//ᵒ⇒comm : (∀ x y → x \\ y ≈ y // x) → Commutative _∙_
```

Comment:
Why is there an `ᵒ` in the name? I think it would work without it?

Reply:
Hmmm... well, formally, it's not the case that the two operations are extensionally equal; there is a 'flip' which should/might (?) be accounted for... but for sure I'm maybe too wedded to the `ᵒ` modifier to connote `flip`... again, I'm not especially committed to this, but equally, I think it's hard to find a good mnemonic/memorable name for things like this in the first place, anyway...

Reply:
Right - I think spelling out 'flip' here would be best.

Reply:
Done.